src/main/webapp/i18n/en/textExercise.json

```
@@ -29,7 +29,18 @@
             "checkPlagiarism": "Check Plagiarism",
             "exampleSubmissionsRequireExercise": "To create an example submission, you first need to finish creating the text exercise. You will be redirected automatically.",
             "saveSuccessful": "Your answer was saved successfully!",
-            "error": "An error occurred. Please try again later."
+            "error": "An error occurred. Please try again later.",
+            "clusterStatistics": {
+                "title": "Cluster Statistics",
+                "description": "A text cluster represents a group of text blocks from student submissions that are identified to be similar.<br/> Once a text block gets a feedback by a tutor, the feedback gets propagated to the rest of text blocks as a suggestion.<br/> The following table shows an overview of all generated text clusters for this exercise.<br/> You can view how much automatic feedback suggestion reuse there is within a cluster. You can also disable a cluster if the similarity grouping has not been performed as expected.",
```

I really like that there is a description now!
Suggestion: "Once there is tutor feedback on a text block, this feedback is propagated to all other text blocks in that cluster. [...] You can also disable a cluster, for example in case a low usage ratio of automatic feedback indicates, that the clustering might not have lead to suggesting reusable feedback during text exercise assessment" 
Because:
- "a" feedback it not very suitable I guess, as feedback is not countable
- "to the rest of the text blocks" is too unspecific 
- "You can also disable a cluster if the similarity grouping has not been performed as expected." is hard to understand.

Maybe you agree :)

---

I agree. Have overlooked it. You may also apply a commit suggestion here, and I could apply it directly 😄

---

So if you agree, you can commit this:
```suggestion
                "description": "A text cluster represents a group of text blocks from student submissions that are identified to be similar.<br/> Once there is tutor feedback on a text block, this feedback is propagated to all other text blocks in that cluster.<br/> The following table shows an overview of all generated text clusters for this exercise.<br/> You can see how many of the automatic feedback suggestions were reused within a cluster. You can also disable a cluster, for example, if a low usage rate of automatic feedback indicates that clustering has not resulted in reusable feedback being suggested during text exercise assessment.",
```